src/easyscore.js

```diff
@@ -271,12 +292,18 @@ export class EasyScore {
     this.options = Object.assign({
       factory: null,
       builder: null,
+      commitHooks: [
+        setId,
+        setClass,
+        Articulation.easyScoreHook,
```

Comment:
While I do agree that a module should be cohesive, I don't agree that the registration of a default commit hook satisfies this.

How does someone reading the code identify what the default `EasyScore` behavior is? When discovering a JS library/class I typically look for some sort of centralized configuration object -- that feels much more intuitive and discoverable. 

But this also leads me to questioning the API, because `EasyScore.addCommitHook` is append only. Don't we need to support completely overriding the defaults? VexFlow provides a basic set, but I easily might want my own set of more specialized commit hooks. What if instead of an `addCommitHook` method we provide a `options.commitHooks` array on construction which defaults to a `static` `EasyScore.defaultCommitHooks`?